client/wc-api/reports/items/selectors.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { isNil } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getResourceName } from '../../utils';
+import { DEFAULT_REQUIREMENT } from '../../constants';
+
+const getReportItems = ( getResource, requireResource ) => (
+	type,
+	query = {},
+	requirement = DEFAULT_REQUIREMENT
+) => {
+	const resourceName = getResourceName( `report-items-query-${ type }`, query );
+	const data = requireResource( requirement, resourceName ) || {};
+
+	return data;
+};
+
+const isGetReportItemsRequesting = getResource => ( type, query = {} ) => {
```

Comment:
I noticed we use `isGetReportItemsRequesting` here while in `client/wc-api/reports/stats/selectors.js` we use `isReportStatsRequesting`, notice one includes `Get` in the name while the other doesn't. I think it would probably be a good idea to unify that and either remove `Get` from `isGetReportItemsRequesting` or add it to `isReportStatsRequesting`.

Reply:
I'll remove "get" from the error and requesting selectors. In the future this will all be DRYed up. 👍

Reply:
Done in 7f934aa.